src/Altinn.Apps/AppFrontend/react/.eslintrc.json

```diff
@@ -176,6 +176,7 @@
     ],
     "react/jsx-boolean-value": "off",
     "linebreak-style": "off",
-    "import/no-cycle": "off"
+    "import/no-cycle": "off",
```

Comment:
I wonder if we really want to disable this check.
Cycle imports can cause crashes. Maybe we could break the files where this is an issue into smaller files?

Reply:
Since this rule was already off, I won't make any changes to this now, but I agree that we should have a discussion about this rule. Created an issue for reviewing/updating our current linter settings: https://github.com/Altinn/altinn-studio/issues/6401